src/lib/select/core/select-utils.ts

```diff
@@ -15,7 +15,7 @@ export function isSelectOptionType(options: ISelectOption[] | ISelectOptionGroup
 }
 
 export function isOptionGroupObject(o: ISelectOption | ISelectOptionGroup): o is ISelectOptionGroup {
-  return isDefined(o) && isObject(o) && o.hasOwnProperty('options') && o.hasOwnProperty('text');
+  return isDefined(o) && isObject(o) && o.hasOwnProperty('options');
```

Comment:
We do not need to ensure that the `text` property is provided since the existing of the `options` property is all we actually care about to determine if this is an option group.